README.md

```diff
@@ -73,15 +73,12 @@ You can go with just the defaults.
       Debian packages from.
       You might want to set this to the address of a local apt proxy
       like *approx*, *apt-cacher* or the like.
-* `docker_cron_image_prune`:
+* `docker_image_prune`:
     * Default: `false`
-    * Description: Cleanup old images without a container reference.
-      A cronjob is created which calls docker's image prune function.
-* `docker_cron_image_prune_syslog_topic`
-    * Default: `docker-prune`
-    * Description: Topic for syslog messages generated by the cronjob.
-      In case of an error an e-mail to *root* will be generated,
-      otherwise the messages are posted to the syslog.
+    * Description: Enables a systemd timer to automatically prune Docker
+      images without container references. You can check the timer with
+      `sudo systemctl list-timers --all` and the results with
+      `sudo journalctl -xeu docker-image-prune`.
```

Comment:
do the log entries produced by systemd timers still show up in syslog? will they have the same `syslog_topic`?